dts/bindings/pwm/nordic,nrf-sw-pwm.yaml

```diff
@@ -5,19 +5,32 @@ compatible: "nordic,nrf-sw-pwm"
 include: [pwm-controller.yaml, base.yaml]
 
 properties:
+    rtc-instance:
+      type: int
+      description: RTC instance to use for generating the PWM output signals
+      required: false
+
     timer-instance:
       type: int
```

Comment:
I see that this already uses an instance number, but I think it would be more appropriate to use phandles.

I haven't reviewed the code in detail, though. I am not putting a -1 or a +1 as a result, just offering a quick comment. I will review further when I can, just wanted to have something to turn around for the time zone difference.

Reply:
I agree, that using phandles is better. The reason why I went with `rtc-instance` is that I wanted to solve one issue at a time, which is adding RTC support to generate PWM. If such conversion to phandle is straightforward to do and you can point to some other driver which resolves RTC/TIMER instance number based on phandle, then I can update this PR. If that is not so simple, then I prefer to convert `rtc-instance` (`timer-instance`) integer to `rtc` (`timer`) phandle in another PR.